### ISSUE TITLE
RepLKNet's code to visualize ERF is used by its results are not compared with

### ISSUE THREAD

**Issue by DingXiaoH** [MzeroMiko/VMamba#2]
Figure 5 uses RepLKNet's code to visualize the ERF.

I appreciate that the authors mentioned in the code that the ERF-related code was copied from RepLKNet.

in https://github.com/MzeroMiko/VMamba/blob/main/analyze/get_erf.py
```
# copied from https://github.com/DingXiaoH/RepLKNet-pytorch
def analyze_erf(source="/tmp/erf.npy", dest="heatmap.png", ALGRITHOM=lambda x: np.power(x - 1, 0.25)):
    # A script to visualize the ERF.
    # Scaling Up Your Kernels to 31x31: Revisiting Large Kernel Design in CNNs (https://arxiv.org/abs/2203.06717)
    # Github source: https://github.com/DingXiaoH/RepLKNet-pytorch
    # Licensed under The MIT License [see LICENSE for details]
```

However, in the caption of Figure 5 it is claimed that "only DeiT and the proposed VMamba" exhibit a global ERF", ignoring that RepLKNet also has a global ERF. And when reporting the results on ImageNet/COCO/ADE20K, RepLKNet is not compared with.

In a nutshell, I wonder if it is appropriate to use RepLKNet's code without mentioning RepLKNet's results (also global ERF) or comparing the results on ImageNet/COCO/ADE20K with RepLKNet.

**Comment by MzeroMiko** [comment]
Sorry for that. We will mention your work in readme.md and cite it in the future.  The reason why we do not compare our work is that replknet does not release tiny (~30M) or small (~50M) models, and all the ERF figs are tested in tiny level (~30M parameters). Also, this work is in progress, we'll cite replknet in the future.
